<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarProductoWindow</class>
 <widget class="QWidget" name="EditarProductoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Producto</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>620</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(2, 2, 2);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:0.339795 rgba(255, 0, 0, 255), stop:0.339799 rgba(255, 255, 255, 255), stop:0.662444 rgba(255, 255, 255, 255), stop:0.662469 rgba(0, 0, 255, 255), stop:1 rgba(0, 0, 255, 255));
background-color: rgb(144, 167, 200);</string>
   </property>
   <widget class="QLabel" name="labelIDProducto">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID del Producto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIDProducto">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>ID generado automáticamente</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProveedor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Proveedor:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxProveedor">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombreProducto">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre del Producto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNombreProducto">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Ejemplo: Air Max, Ultraboost</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescripcion">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Descripción:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDescripcion">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Descripción del producto...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTalla">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Talla Disponible:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTalla">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Ejemplo: 25, 26, 27, 28</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrecio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Precio por Unidad:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrecio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Ejemplo: 1500.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCategoria">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Categoría:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCategoria">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFechaSalida">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fecha de Salida:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditFechaSalida">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>12</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>12</month>
      <day>2</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(255, 255, 255);
    border-radius: 15px;
    border: 1px solid rgb(144, 167, 200);
}

QPushButton:hover{
    background-color: rgb(227, 227, 227);
    border: 1px solid rgb(120, 140, 180);
}</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>560</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(255, 255, 255);
    border-radius: 15px;
    border: 1px solid rgb(144, 167, 200);
}

QPushButton:hover{
    background-color: rgb(227, 227, 227);
    border: 1px solid rgb(120, 140, 180);
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="regresar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>
        QLabel {
            font-family: &quot;Arial Rounded MT Bold&quot;;
            font-size: 16pt;
            border-radius: 10px;
        }
        QLabel:hover {
            background-color: #ccc;
        }
        QLabel:pressed {
            background-color: #aaa;
        }
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../flecha.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
